docker/venice-controller/single-dc-configs/controller.properties

```diff
@@ -0,0 +1,37 @@
+admin.port=5555
+admin.secure.port=5556
+zookeeper.address=zookeeper:2181
+kafka.zk.address=zookeeper:2181
+kafka.bootstrap.servers=kafka:9092
+controller.name=venice-controller
+cluster.name=venice-cluster
+storage.engine.overhead.ratio=0.85
+ssl.to.kakfa=false
+controller.parent.mode=false
+controller.system.schema.cluster.name=venice-cluster
+cluster.to.d2=venice-cluster:venice-discovery
+delay.to.rebalance.ms=0
+offline.job.start.timeout.ms=60000
+topic.cleanup.sleep.interval.between.topic.list.fetch.ms=100
```

Comment:
a thought for future versions: 
Should add some explanation on these configs (or point to the documentation which has these details) such that the user who is trying to explore this knows what are set and how modifying them will affect the runtime.

Reply:
Yeah